config/lint.xml

```diff
@@ -15,7 +15,9 @@
     <issue id="InconsistentArrays" severity="error" />
     <issue id="HardcodedText" severity="error" />
     <issue id="IconDuplicates" severity="error" />
-    <issue id="IconDipSize" severity="error" />
+    <issue id="IconDipSize" severity="error">
+        <ignore path="**/libs/editor/src/main/res/drawable-*/format_bar_chevron.png" />
+    </issue>
```

Comment:
Is this something we plan to fix soon? (but in a separate PR though)

Reply:
This is one of the many warnings that we are having, as such I wouldn't bet on it being planned to be fixed soon. You will notice that I just moved this warning from one place (by removing the `editor` related `lint.xml` file) to this common place (see [commit](https://github.com/wordpress-mobile/WordPress-Android/pull/16769/commits/bb4a2d775abd1e2d5387ae9027dd676ecdac8580) description).

As such, I wouldn't worry about fixing this warning, at least as part of this group of PRs or in general, this project. This kind of warnings should be fixed as part of a more generic Lint project. Hopefully, we will get to that later on and sooner than later. 🤞

Does that makes sense @AliSoftware ?